.github/workflows/validate-windows-ut.yml

```diff
@@ -0,0 +1,15 @@
+name: Windows Powershell Unit Test
+on: push
+
+jobs:
+  pester-test:
+    name: Pester test
+    runs-on: windows-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Perform a Pester test from the tests.ps1 file
+        shell: pwsh
+        run: |
+          Invoke-Pester parts/windows/*.tests.ps1 -Passthru
+          Invoke-Pester staging/cse/windows/*.tests.ps1 -Passthru
```

Comment:
Two ways to select here:
1. Only run staging/cse/windows/cse.tests.ps1
2. Run all *.tests.ps1 files and remove cse.tests.ps1 file?

Reply:
I think that you can remove cse.tests.ps1 since we will have a unit test pipeline and avoid to add new ut test file to cse.tests.ps1